invenio_app_ils/ui/src/invenio_app_ils/pages/frontsite/PatronProfile/PatronCurrentLoans/PatronCurrentLoans.js

```diff
@@ -0,0 +1,165 @@
+import React, {Component} from 'react';
+import PropTypes from 'prop-types';
+
+import {
+  Loader,
+  Error,
+  Pagination,
+} from '../../../../common/components';
+import {toShortDate} from '../../../../common/api/date';
+import {invenioConfig} from '../../../../common/config';
+import {
+  Container,
+  Grid,
+  Header,
+  Item,
+  Label, Message,
+} from "semantic-ui-react";
+import {getCover} from "../../config";
+import {Link} from "react-router-dom";
+import {FrontSiteRoutes} from "../../../../routes/urls";
+import {DocumentAuthors} from "../../components/Document";
+import isEmpty from "lodash/isEmpty";
+import { DateTime } from "luxon";
+import {ILSItemPlaceholder} from
+    "../../../../common/components/ILSPlaceholder/ILSPlaceholder";
+
+
+class LoanListEntry extends Component {
+
+  render() {
+    const {loan} = this.props;
+    const now = DateTime.local();
+    const is_loan_overdue = loan.metadata.end_date < now;
+    return (
+      <Item className={is_loan_overdue ? 'bkg-danger' : ''}
+            key={loan.metadata.pid}>
+        <Item.Image size='mini'
+                    src={getCover(loan.metadata.document_pid)}
+                    as={Link}
+                    to={FrontSiteRoutes.documentDetailsFor(
+                      loan.metadata.document_pid)}
+        />
+
+        <Item.Content>
+          <Item.Header as={Link}
+                       to={FrontSiteRoutes.documentDetailsFor(
+                         loan.metadata.document_pid)}>
+            {loan.metadata.document.title}</Item.Header>
+          <Grid columns={2}>
+            <Grid.Column mobile={16} tablet={8} computer={8}>
+              <Item.Meta>
+                <DocumentAuthors metadata={loan.metadata.document}/>
+                Loaned on {toShortDate(loan.metadata.start_date)}
+              </Item.Meta>
+              <Item.Description>
+                {}
+                You have extended this loan {loan.metadata.extension_count}
+                {' '}of {invenioConfig.loans.max_extensions_count} times
+              </Item.Description>
+            </Grid.Column>
+            <Grid.Column textAlign={'right'}
+                         mobile={16} tablet={8} computer={8}>
+              <Item.Description>
+                Please return the literature before date{' '}
+                <Label className={'bkg-warning'}>
```

Comment:
Minor comment about the label bg color: I think it might look better if `bkg-warning` was more towards red for overdue loans but as it is now if it's not overdue. What do you think?

Reply:
in general it is orange, primary color of the frontsite and I wanted the date to be highlighted, maybe I'll just change the className in this case as it might be confusing